src/components/atoms/Select.tsx

```diff
@@ -253,9 +253,15 @@ export const Select: React.FC<SelectProps> & {
   return (
     <SelectContext.Provider value={{ value, onChange, setLabel }}>
       <SelectContainer className={className} ref={containerRef} width={width}>
-        <input type="text" readOnly value={value} style={{ display: 'none' }} {...props} />
+        <input
+          type="text"
+          readOnly
+          value={value === undefined ? '' : value}
+          style={{ display: 'none' }}
+          {...props}
+        />
         <SelectButton error={error} value={value} onClick={() => setOpened((prev) => !prev)}>
-          {((value !== null || value !== undefined) && label) ?? placeholder}
+          {value === null || value === undefined ? placeholder : label}
```

Comment:
P5: `Lodash`에 `isNil`이 같은 기능을 하더라구요. 코드가 짧아지기는 하는데 외부 라이브러리를 쓰는게 좋은건지는 모르겠어용

```suggestion
          {_.isNil(value) ? placeholder : label}
```

Reply:
오.. lodash를 어차피 사용하고 있는 라이브러리기도 하고 많이 사용하는 라이브러리니까 괜찮을 것 같아요!!